### PR TITLE
DeviceItem: get icons from Pulse

### LINKED ISSUE #222: Change icon of unused outputs/inputs to muted variants
### Problem

If you have more than one audio output/input connected and are using only one, the popover will show audio icons next to every output/input, which makes it look like they're all being used at the same time.
Screenshot from 2021-10-19 17-35-48

(Sonata Audio is my USB DAC that my headphones are plugged into)

### Proposal

Change the audio icons of each output/input not being used to audio-volume-muted. This makes it more clear which output/input is being used, and it doesn't look like they're all being used at once.

### Prior Art (Optional)

_No response_

### PR DESCRIPTION
**Pull request by danrabbit**
Same method as https://github.com/elementary/switchboard-plug-sound/pull/213

Maybe Fixes #222 ?

screenshot

### CODE REVIEW

**Comment by D0J0P** [comment]
Regarding #222, if there are going to be more icons for inputs and outputs that don't have muted variants, would greying them out be reasonable?

**Comment by danrabbit** [comment]
@D0J0P I'm not really sure that's necessary anymore with this design change. I don't think we'd want to make a device look like it is unavailable

File: data/sound.appdata.xml.in [hunk 1]
```diff
@@ -7,6 +7,15 @@
   <summary>Adjust speaker and microphone volume or see Now Playing information and media controls</summary>
   <icon type="stock">preferences-desktop-sound</icon>
   <releases>
+    <release version="6.0.1" date="2021-11-08" urgency="medium">
+      <description>
+        <p>Minor updates</p>
+        <ul>
+          <li>Get icons for more device types</li>
```
Comment (cassidyjames) [inline data/sound.appdata.xml.in:14]: Just a little more end-user focused:

```suggestion
          <li>Show more specific icons for different types of devices</li>
```

**Review by cassidyjames — APPROVED** [review]
Nice improvement like the update to the plug design. :+1: If we want to think about mute icons, we can do so in a follow-up, but I agree that showing the specific device type icons here makes #222 less of an issue.

**Pull request merged**